sw/device/rom_exts/rom_ext_test_base.c

```diff
@@ -0,0 +1,24 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdbool.h>
+#include <stddef.h>
+
+#include "sw/device/lib/runtime/hart.h"
+
+int main(int argc, char *argv[]);
+
+#include "sw/device/rom_exts/rom_ext.h"
```

Comment:
Uh. Why is this `#include` not at the top of the file?

Reply:
This file has been deleted.

Done.